runtimes/alibi-explain/mlserver_alibi_explain/common.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+from asyncio import AbstractEventLoop
+from enum import Enum
+
+#from mlserver_alibi_explain.explainers.anchor_image import AnchorImageWrapper
+#from mlserver import MLModel
+from typing import Any, Dict, Optional, Type, Callable, Coroutine, Awaitable
+
+import numpy as np
+import requests
+from pydantic import BaseSettings
+
+from mlserver.types import ResponseOutput, InferenceResponse, InferenceRequest
+
+_ANCHOR_IMAGE_TAG = 'anchor_image'
+
+ENV_PREFIX_ALIBI_EXPLAIN_SETTINGS = "MLSERVER_MODEL_ALIBI_EXPLAIN_"
+
+
+class ExplainerEnum(str, Enum):
+    anchor_image = _ANCHOR_IMAGE_TAG
+
+
+# TODO: is this inference response?
+def create_v2_from_any(data: Any, name: str) -> Dict:
```

Comment:
Should we wrap this into a Codec? Conscious that it's something kind of temporary, but could be maybe useful in other cases?

Reply:
I agree, these are the type of components that we'll end up also needing in Tempo if we want to easily create clients that can convert to/from explainers